### PR TITLE
addwebapp-crud

### PR DESCRIPTION
**Pull request by dev-Architec**

### CODE REVIEW

File: Api/Mapper/MappingConfig.cs [hunk 1]
```diff
@@ -8,6 +8,8 @@ public class MappingConfig : Profile
     {
         public MappingConfig()
         {
+
+            CreateMap<ApplicationUser, UserResponse>();
```
Comment (aquarianchd83) [inline Api/Mapper/MappingConfig.cs]: why you added the code in the api mapper file ?

File: WebApp/Controllers/UserController.cs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            LastName = user.LastName,
+            PhoneNumber = user.PhoneNumber,
+            Name = user.Name,
+            Email = user.Email,
+        };
+        return View(model);
+    }
+
+
+    public async Task<ActionResult> Delete(string id)
```
Comment (aquarianchd83) [inline WebApp/Controllers/UserController.cs:73]: why we created this action ?

File: Api/Mapper/MappingConfig.cs (hunk 1 above)
Comment (dev-Architec) [reply Api/Mapper/MappingConfig.cs]: i will remove this

File: WebApp/Controllers/UserController.cs (hunk 2 above)
Comment (dev-Architec) [reply WebApp/Controllers/UserController.cs:73]: for delete functionality

**Pull request merged**